snewpdag/plugins/gen/TimeSeries.py

```diff
@@ -18,6 +18,15 @@
 from snewpdag.dag import Node
 from . import TimeDistSource
 
+#rdallava: function used to insert a dictionary into a tuple
+def add_dictionary_to_a_tuple(dictionary):
```

Comment:
should the original tuple be an argument?

Reply:
Oh there is no original tuple actually. One creates  a new tuple and store the original dictionary inside it. This is because the Combine.py plugin wants to be fed with a tuple and not a dictionary.